isaric/parsers/ccp-netherlands.toml

```diff
@@ -0,0 +1,736 @@
+    { field = "ethnic_group#Latin_America", values = { "1.0" = "Latin American" } },
+    { field = "ethnic_group#White", values = { "1.0" = "White" } },
+    { field = "ethnic_group#AboriginalFirst_Nations", values = { "1.0" = "Aboriginal/First Nations" } },
+    { field = "ethnic_group#Unknown", values = { "1.0" = "Unknown" } },
+  ]
+
+
+[subject.works_microbiology_lab]
+  field = "microbiology_worker"
+  ref = "Y/N"
+
+[subject.works_healthcare]
+  field = "healthcare_worker"
+  ref = "Y/N"
+
+[subject.pregnancy]
+  field = "Pregnancy"
+  ref = "Y/N"
+
+[subject.pregnancy_post_partum]
+  field = "post_partum"
+  ref = "Y/N"
+
+[subject.pregnancy_gestational_age_weeks]
+  field = "gestational_weeks"
+
+[subject.has_chronic_hematologic_disease]
+  field = "chd" # ?
+  ref = "Y/N"
+
+[subject.has_chronic_cardiac_disease]
+  field = "ccd"
+  ref = "Y/N"
+
+[subject.has_chronic_kidney_disease]
+  field = "ckd"
+  ref = "Y/N"
+
+[subject.has_chronic_respiratory_disease]
+  field = "cpd" # pulmonary disease?
+  ref = "Y/N"
+
+[subject.has_dementia]
+  field = "Dementia"
+  ref = "Y/N"
+
+[subject.has_obesity]
+  field = "obesity"
+  ref = "Y/N"
+
+[subject.has_rheumatologic_disorder]
+  field = "rheuma_disorder"
+  ref = "Y/N"
+
+[subject.has_hiv]
+  field = "aids_hiv"
+  ref = "Y/N"
+
+[subject.has_hypertension]
+  field = "baseline_hypertension"
+  ref = "Y/N"
+
+[subject.has_malignant_neoplasm]
+  field = "mneoplasm"
+  ref = "Y/N"
+
+[subject.has_smoking]
+  field = "Smoking"
+
+  [subject.has_smoking.values] # these 3 are present, who knows if the mapping is correct
+    1 = "yes"
+    2 = "former"
+    3 = "never"
+
+[subject.has_asthma]
+  field = "asthma"
+  ref = "Y/N"
+
+[subject.has_diabetes]
+  combinedType = "any"
+  fields = [
+    { field = "diabetes_complications", ref = "Y/N" },
+    { field = "diabetes_without_complications", ref = "Y/N" },
+  ]
+
+[subject.has_liver_disease]
+  combinedType = "any"
+  fields = [
+    { field = "live_disease", ref = "Y/N" },
+    { field = "mld", ref = "Y/N" },
+  ]
+
+[subject.has_malnutrition]
+  field = "Malnutrition"
+  ref = "Y/N"
+
+[subject.has_comorbidity_other]
+  combinedType = "set"
+  excludeWhen = "none"
+  fields = [{ field = "Specif_other_risk_factors" }]
+
+[subject.has_died]
+  field = "Outcome"
+  values = { 4 = true }
+
+[subject.date_death]
+  field = "Outcome_dt"
+  if = { Outcome = 4 }
+
+[subject.icu_admitted]
+  field = "ICU_High_Dependency_admission"
+  ref = "Y/N"
+
+  # VISIT
+  # required fields - "visit_id", "subject_id", "country_iso3", "start_date", "outcome", "date_outcome"
+
+[visit]
+  country_iso3 = "NLD"
+
+  [visit.visit_id]
+    field = "record_id"
+    sensitive = true
+
+  [visit.subject_id]
+    field = "record_id"
+    sensitive = true
+
+  [visit.start_date]
+    field = "admission_dt"
+
+  [visit.pathogen_test_date]
+    field = "collection_dt"
+    if = { Pathogen_tested_Detected_result = { "=~" = "sars-cov-2" } }
```

Comment:
interesting. 
is "=~" an existing logical condition? or a typo?

Reply:
It's a new operator Abhishek recently added, to allow for regex string searching (in this case it means it matches the string case-insensitively so Sars-COV-2, sars-Cov-2 etc all match)